scripts/update-repo-internal.sh

```diff
@@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
 case $PARENT in
     centos)
         mkdir -p repo/$PARENT/$RELEASE/packages
-        find package/$DISTRO/ -name *.rpm | \
+        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \
```

Comment:
Sorry that is a typo, I was editing with vi and didn't realize I was already in insert mode hence the 'i'.

--Matt

----- Original Message -----
From: "Andrew Melo" <notifications@github.com>
To: "accre/lstore-release" <lstore-release@noreply.github.com>
Sent: Sunday, November 8, 2015 5:52:07 PM
Subject: Re: [lstore-release] Rebased version of vandy-cms branch (#24)



In scripts/update-repo-internal.sh : > @@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
>  case $PARENT in
>      centos)
>          mkdir -p repo/$PARENT/$RELEASE/packages
> -        find package/$DISTRO/ -name *.rpm | \
> +        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \ 

Why i'*.rpm' and not '*.rpm' ? 

— 
Reply to this email directly or view it on GitHub .